modules/apps/remote-app/remote-app-admin-web/src/main/java/com/liferay/remote/app/admin/web/internal/application/list/RemoteCustomElementAdminPanelApp.java

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
A small commit, +11,289 lines. I'm going to assume than most of these are close to remote apps implementation.

Reply:
Yes, it's copy and paste with changes to account for the new field.